k8s/k8s.go

```diff
@@ -761,6 +776,7 @@ type k8sOps struct {
 	config             *rest.Config
 	dynamicInterface   dynamic.Interface
 	ocpClient          ocp_clientset.Interface
+	ocpSccClient       ocp_scc_clientset.Interface
```

Comment:
nit: Can we call this `ocpSecurityClient` instead of SccClient as it doesn't apply to only SCCs?

Reply:
Done